sync2jira/upstream_issue.py

```diff
@@ -346,6 +346,11 @@ def github_issues(upstream, config):
         yield issue
 
 
+def get_github_project_details(graphqlurl, headers, ghquery, variables):
+    response = requests.post(graphqlurl, headers=headers, json={"query": ghquery, "variables": variables})
+    return response
```

Comment:
I don't see any benefit to encapsulating the call to `post()` inside a helper function.  Why did you do this?

If it is just for mocking, it's as easy to mock `requests.post()` as it is to mock `get_github_project_details()`.  There is [a nice example here](https://github.com/release-engineering/Sync2Jira/blob/18dce8e31431fef4f484e2eab9cbc70f5088a2b6/tests/test_main.py#L510-L513) (albeit with `get()` instead of `post()`, but it works the same).

Reply:
Fixed in 725a04868080b4dd7aebced4a40a6b28bb8725e5